web/blog/tenzir-v4.1/index.md

```diff
@@ -0,0 +1,146 @@
+and retro matching.
+
+[sigma-docs]: https://docs.tenzir.com/next/operators/transformations/sigma
+[sigma-github]: https://github.com/SigmaHQ/sigma
+[unified-detection]: https://docs.tenzir.com/use-cases/unified-detection
+
+## Show Operator
+
+The experimental [`show` operator][show-docs] supersedes the `version` operator.
+Use `show <aspect>` to show various aspects of a Tenzir node. The following
+aspects are currently available:
+
+- `build`: shows compile-time build information.
+- `config`: shows all current configuration options.
+- `connectors`: shows all available [connectors][connectors-docs].
+- `dependencies`: shows information about build-time dependencies.
+- `fields`: shows all fields of existing tables at a remote node.
+- `formats`: shows all available [formats][formats-docs].
+- `operators`: shows all available [operators][operators-docs].
+- `partitions`: shows all table partitions of a remote node.
+- `pipelines`: shows all managed pipelines of a remote node.
+- `plugins`: shows all loaded plugins.
+- `types`: shows all known types at a remote node.
+- `version`: shows the Tenzir version.
+
+This enables powerful introspection use-cases. Here are some examples that we
+found useful at Tenzir.
+
+Show all running pipelines with an ingress of over 10 MiB:
+
+```
+show pipelines
+| where total.ingress.num_approx_bytes > 10 Mi
+```
+
+Show the memory usage of the node's catalog by schema in descending order:
+
+```
+show partitions
+| summarize memory_usage=sum(memory_usage) by schema
+| sort memory_usage desc
+```
+
+Print the configured endpoint of the node (returns `null` if not customized):
+
+```
+show config
+| put tenzir.endpoint
+```
+
+[show-docs]: https://docs.tenzir.com/operators/sources/show
+[connectors-docs]: https://docs.tenzir.com/connectors
+[formats-docs]: https://docs.tenzir.com/formats
+[operators-docs]: https://docs.tenzir.com/operators
+
+## Compress and Decompress Operators
+
+The [`compress`][compress-docs] and [`decompress`][decompress-docs] operators
+make it easy to read and write compressed data. The operator uses Apache Arrow's
+compression utilities under the hood, and transparently supports all options
+that Apache Arrow supports for streaming compression. The currently supported
+codecs are `brotli`, `bz2`, `gzip`, `lz4`, and `zstd`.
+
+For example, the following pipeline creates a Gzip-compressed NDJSON file that
+contains all events that were previously imported at the node:
+
+```
+export
+| write json --compact-output
+| compress gzip
+| save file /tmp/backup.json.gz
+```
+
+[compress-docs]: https://docs.tenzir.com/operators/transformations/compress
+[decompress-docs]: https://docs.tenzir.com/operators/transformations/decompress
+
+## Small Things
+
+We're constantly polishing the node's pipeline execution engine, and improving
+the app's usability. Since the last release, we've improved pipeline execution
+to smake low pipelines return their initial result faster, and have improved the
```

Comment:
> smake

Reply:
make slow, not sure how that typo happened 🫠 Will fix, thanks!